packages/styles/src/core.scss

```diff
@@ -2,7 +2,4 @@
 @forward './variables';
 @forward './mixins';
 
-@use './lic/bootstrap-license';
-@use './placeholders';
```

Comment:
Are you sure, every component imports the dependant placeholders by itself?

Reply:
If this is not the case, it's a mistake. According to the test files where every component is rendered individually, all components import their respective placeholders.